docs/docs/home/home.md

```diff
@@ -1,62 +1,124 @@
 
-If you're looking for an in-depth tutorial that walks you through our available features, click [here](/guides/tutorial/rill-basics/launch)! 
+### **GitHub Analytics**: 
 
-Some tutorials include:
+Analyze GitHub activity to understand what parts of your codebase are most active, analyze contributor productivity, and evaluate the intersections between commits and files
 
-- [From Source to Deployment in 6 Steps!](/guides/tutorial/rill-basics/launch): Data import, transformation, dashboard creation, and deployment!
-- [Rill and ClickHouse](/guides/tutorial/rill-clickhouse/): For our friends coming from ClickHouse, get started in **5 steps**.
+- <a href="https://github.com/rilldata/rill-examples/tree/main/rill-github-analytics">GitHub →</a><br />
+- <a href="/guides/github-analytics">Walkthrough →</a><br />
+- <a href="https://ui.rilldata.com/demo/rill-github-analytics">Live Demo →</a>
 
-The final project is available on our [example repository](https://github.com/rilldata/rill-examples/tree/main/my-rill-tutorial). 
+### **App Engagement**
 
+A conversion dataset used by marketers, mobile developers, or product teams to analyze funnel steps
 
+- <a href="https://github.com/rilldata/rill-examples/tree/main/rill-app-engagement">GitHub →</a><br />
+- <a href="https://ui.rilldata.com/demo/rill-app-engagement">Live Demo →</a>
 
-## Release Notes
+### **Kitchen-sink example**: 
 
-Curious about what's new? Check out our latest and greatest updates in the [release notes!](https://docs.rilldata.com/notes)
+A compilation of projects that deep dives in Rill's features using  ClickHouse's GitHub commit information.
 
-## Don't be a stranger!
-We always appreciate hearing from our users and welcome any questions, comments, or concerns you may have. To get in touch with us, there are a few options available:
-1. _**File an issue**_: Spotted a bug or have a brilliant idea? Let us know by [filing an issue](https://github.com/rilldata/rill/issues/new/choose) on GitHub.
-2. _**Join the party on Discord!**_ Whether you have questions or need a hand getting started, our community and Rill team members are here to help. Join us on [Discord](https://discord.gg/DJ5qcsxE2m).
-3. _**Contact Rill Support**_: If you are an existing prospect or customer, you can also reach [Rill Support](/contact#contacting-support) through Slack, email, or in-app chat. 
+- <a href="https://github.com/rilldata/rill-examples/tree/main/my-rill-tutorial">GitHub →</a><br />
+- <a href="/guides/rill-basics/launch">Walkthrough →</a><br />
+- <a href="https://ui.rilldata.com/demo/my-rill-tutorial">Live Demo →</a>
 
 
-:::info
+## Explore Rill's Capabilities
 
-Please make sure to abide by our [community policy](https://github.com/rilldata/rill/blob/main/COMMUNITY-POLICY.md)!
+<div className="tile-icon-grid">
+<TileIcon
+  header="Connect Data Sources"
+  content="Connect to your data sources and start ingesting data into Rill for analysis."
+  link="/reference/connectors/"
```

Comment:
All of these tiles have links with trailing slashes. Ideally, we remove the trailing slash everywhere.